.env.sample

```diff
@@ -1,4 +1,4 @@
-TESTNET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
-TESTNET_CHAIN_URL=url to connect to testnet
+STAGENET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
```

Comment:
We are trying to be more careful about how we use naming. `STAGENET` is how we are now referring to the poa network we currently have running on AWS